topics/ui/controls/combo_box.md

```diff
@@ -23,80 +23,119 @@ comboBox.setEditable(true);
 
 ## When to use
 
-Use a combo box if:
 
-* The user needs to select a value or enter a custom value.
+### Select value or enter custom
 
-  ![](combo_box_font_size.png){width=165}
+When the user needs to select a value or enter a custom value, use the combo box:
 
-* The possible choices are objects or states.
+![A font size  that allows selecting predefined values as well as entering a custom value](combobox_when_to_use_1.png){width=706}
 
+### Return to previous states
+If the user may need to return to previous values, use the combo box. For example, save previously entered paths, so the user can quickly select them later:
```

Comment:
I think this could be shortented a bit. ChatGPT suggested this version "Use a combo box if users may need to revisit previous values, such as saving entered paths for quick selection later:"